### ISSUE TITLE
Cards: Pass To...

### ISSUE THREAD

**Issue by pemerson71** [MetaMorphic-Digital/torgeternity#239]
This is something I forgot to point out was happening in May, when I began running the original Torg Destiny Map adventure... I should have pointed it out, then, but it was a long night and I had forgotten about it by the end of the night.  This is more a suggestion than any sort of trouble ticket, but I thought it might be more important to put it here, so you guys can kill it or get to it, eventually.

While I was teaching three of my new Players how to use the updated card system ya'll built, we came across the Pass To... function for the cards.  The drop-down list has ALL of the stacks listed on it, not just PCs.  Would it be possible to somehow exclude those other decks, so only Player-Character card hands and pools are shown, please?  If a Pool card is being traded, the Pass To... could take from the Card Pool and deliver to the Card Pool of the other but, if it's the Hand being traded to and from, from the Hand of one and delivered to the Hand of the other, please?

I know I'm part of the team, but I am amazed by the work you guys do all the time.  Thank you.

**Comment by Durak-Git** [comment]
I tested some filtering.
The **pass to** is for players hand exchange, **discard** and **play** transfer to appropriate discard piles.
So I limited the player's choice to **_other active players_**.
I keep the full list if GM needs to do the **pass to** to a non connected player
(do not pay attention to my fantastic character's name in the following screenshots)
2 players and a GM connected
the gm view
![GmView](https://github.com/gmmatt/torgeternity/assets/97435195/03dab14d-0880-4d84-a403-aa9a98693bfc)

player SK1 view 
![P1View](https://github.com/gmmatt/torgeternity/assets/97435195/2ad2ad21-3887-49ce-88f6-06e2bc748185)

player ghgh view
![P2View](https://github.com/gmmatt/torgeternity/assets/97435195/5379d621-3c83-416e-ad1c-2ebdd97fc7c8)

Do I modify it like this ?
Still opened to different choices

**Comment by pemerson71** [comment]
Allow me to address this as a GM... Typically, I don't have to use Pass To... at all.  When I go to move cards, perhaps someone has played a Seize Initiative, or mis-played a card to another Player, but got the Player name wrong -which, I believe, is the original reason I wrote this up- I typically just have to open one or two stacks and move a card from one stack to the other.  By the way, the ability to move cards from one stack to another, where it's removed from the exiting stack and added to the appropriate stack, is pure freakin' genius, as far as I'm concerned.

So, in short, I don't think many GMs will use Pass To..., unless they're playing a Storm Knight NPC, of course.  However, Players do need to have Pass To... displaying ONLY active Players/Characters, if possible, please?

**Comment by Algor0113** [comment]
For now, issue closed. Players do have the appropriate stacks displayed, the GM should do some clearance on the stacks if there are stacks, that are no longer needed. As GM, you _must_ have a complete overview.
